src/types/classes/invoice.ts

```
@@ -0,0 +1,384 @@
+	public ShipMethodRef?: ReferenceType;
+
+	/**
+	 * @description Billing address
+	 */
+	public BillAddr?: PhysicalAddress;
+
+	/**
+	 * @description Tax calculation order
+	 * @description true = apply discount before tax
+	 */
+	public ApplyTaxAfterDiscount?: Boolean;
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)